src/fin.ts

```diff
@@ -0,0 +1,946 @@
+        },
+        {
+          name: "yml",
+          description:
+            "Display static YML project config suitable for export (NOTE: SECRET_* values will not be hidden)",
+        },
+        {
+          name: "generate",
+          description: "Generate empty Docksal configuration for the project",
+          options: [
+            {
+              name: "--stack",
+              description:
+                "Set non-default DOCKSAL_STACK during config generate",
+              requiresEquals: true,
+              args: {
+                name: "stack",
+              },
+            },
+            {
+              name: "--docroot",
+              description: "Set non-default DOCROOT during config generate",
+              requiresEquals: true,
+              args: {
+                name: "directory",
+              },
+            },
+          ],
+        },
+        {
+          name: "set",
+          description: "Set value(s) for the variable(s) in project ENV file",
+          options: [
+            {
+              name: "--global",
+              description: "Set for global ENV file",
+            },
+            {
+              name: "--env",
+              description: "Set in environment specific project ENV file",
+              requiresEquals: true,
+              args: {
+                name: "name",
+              },
+            },
+          ],
+          args: {
+            name: "VAR=VAL",
+            isVariadic: true,
+          },
+        },
+        {
+          name: ["remove", "rm"],
+          description: "Remove variable(s) from project ENV file",
+          options: [
+            {
+              name: "--global",
+              description: "Remove from global ENV file",
+            },
+            {
+              name: "--env",
+              description: "Remove from environment specific project ENV file",
+              requiresEquals: true,
+              args: {
+                name: "name",
+              },
+            },
+          ],
+          args: {
+            name: "VAR",
+            isVariadic: true,
+          },
+        },
+        {
+          name: "get",
+          description:
+            "Get the value of the single variable from project ENV file",
+          options: [
+            {
+              name: "--global",
+              description: "Get value from global ENV file",
+            },
+            {
+              name: "--env",
+              description:
+                "Get value from environment specific project ENV file",
+              requiresEquals: true,
+              args: {
+                name: "name",
+              },
+            },
+          ],
+          args: {
+            name: "VAR",
+          },
+        },
+      ],
+    },
+    {
+      name: "exec",
+      description: "Execute a command or a script in cli",
+      args: {
+        name: "command|file",
+      },
```

Comment:
```suggestion
      args: {
        name: "command|file",
        isCommand: true,
      },
```
Will give correct suggestions if you provide a command name there!